lib/jnpr/junos/utils/config.py

```diff
@@ -766,15 +770,27 @@ def _open_configuration_batch():
         def _open_configuration_exclusive():
             return self.lock()
 
+        def _open_configuration_ephemeral(**kwargs):
+            self.rpc.open_configuration(**kwargs)
+            return True
+
         def _unsupported_option():
             if self.mode is not None:
                 raise ValueError("unsupported action: {0}".format(self.mode))
 
+        if self.kwargs.get('ephemeral_instance'):
+            ephemeral_kwargs = {
+                'ephemeral_instance': self.kwargs['ephemeral_instance']}
+        else:
+            ephemeral_kwargs = {'ephemeral': True}
+
         {
             'private': _open_configuration_private,
             'dynamic': _open_configuration_dynamic,
             'batch': _open_configuration_batch,
-            'exclusive': _open_configuration_exclusive
+            'exclusive': _open_configuration_exclusive,
+            'ephemeral': lambda: _open_configuration_ephemeral(
```

Comment:
Can you explain why lambda is required here, but not in the other cases? Is it because `_open_configuration_ephemeral()` takes arguments?